### PR TITLE
Various typo fixes and a few SQL statement modifications

### PR DESCRIPTION
**Pull request by julsfelic**
Just finished your book which was a nice refresher on SQL and went ahead and added some changes.

### CODE REVIEW

File: aggregation/aggregation_functions.md [hunk 1]
```diff
@@ -97,22 +97,22 @@ We're going to write up a fairly complicated query this time. This query compute
 Our query will look like this:
 
 ```sql
-SELECT Post.avgRating, Pre.avgRating
+SELECT Post.avgRating - Pre.avgRating
```
Comment (hgducharme) [inline aggregation/aggregation_functions.md:100]: I'm not sure what is going on here, I believe this should be a comma. Could you explain why you are changing this to a hyphen?

**Comment by hgducharme** [comment]
1. `README.md` - Good
2. `aggregation_clauses.md` - Good
3. `aggregation_functions.md` - See line note
4. `basic_select_statements/README.md` - Good
5. `subqueries_in_the_from_&_select_clause/README.md` - Good
6. `subqueries_in_the_where_clause/using_operators_in_a_subquery.md` - Good
7. `the_join_family_operators/README.md` - Good
8. `the_join_family_operators/natural_join.md`
9. `the_join_family_operators/outer_join.md` - Good
10. `the_join_family_operators/the_inner_join_operator.md` - Good

**Comment by hgducharme** [comment]
Ahh nevermind, I see what is going on in number 3; all good now. Good work!

**Comment by julsfelic** [comment]
No problemo! Thanks!

**Pull request merged**